app/models/computacenter/outgoing_api/cap_update_request.rb

```diff
@@ -36,6 +36,7 @@ def handle_response!
       )
     end
 
+    SchoolDeviceAllocation.where(id: @allocation_ids).update_all(cap_update_request_timestamp: @timestamp, cap_update_request_payload_id: @payload_id)
```

Comment:
It feels like we're breaking encapsulation a bit by putting this update here, rather than at the point where `post!` is called... In my mind, this class should concern itself with CC request/response logic but shouldn't really know too much about allocations/caps etc.

Reply:
yep, fair point - done in cadfa4e